packages/edit-site/src/components/template-actions/index.js

```diff
@@ -22,7 +23,12 @@ import { store as editSiteStore } from '../../store';
 import isTemplateRemovable from '../../utils/is-template-removable';
 import isTemplateRevertable from '../../utils/is-template-revertable';
 import RenameMenuItem from './rename-menu-item';
-import { TEMPLATE_POST_TYPE } from '../../utils/constants';
+import {
+	TEMPLATE_POST_TYPE,
+	TEMPLATE_PART_POST_TYPE,
+	POST_TYPE_LABELS,
+	PATTERN_TYPES,
+} from '../../utils/constants';
 
 export default function TemplateActions( {
```

Comment:
Does the name of this component still make sense?

Reply:
Not really to me. 

My understanding is long term we'll be moving to everything being "patterns", which includes template parts and templates(?).  I'd lean towards templates being specialised patterns rather than the other way around.

Reply:
Thanks for the confirmation. 

I think we could rename it to `pattern-actions` or something. I'll make a note to do this as a follow up 👍🏻